torch/csrc/distributed/rpc/BuiltinOp.h

```diff
@@ -0,0 +1,32 @@
+#pragma once
+
+#include <torch/csrc/distributed/rpc/Message.h>
+#include <torch/csrc/distributed/rpc/rpc_headers.h>
+#include <vector>
+
+namespace torch {
+namespace distributed {
+namespace rpc {
+
+// A buildin operator
```

Comment:
nit: built-in

Reply:
I was able to easily figure out what this class does, but people who are not so familiar with rpc will not. There should be a more detailed comment saying what this class represents.